flows/msg.go

```diff
@@ -27,7 +27,8 @@ type MsgIn struct {
 type MsgOut struct {
 	BaseMsg
 
-	QuickReplies_ []string `json:"quick_replies,omitempty"`
+	QuickReplies_ []string       `json:"quick_replies,omitempty"`
+	Templating_   *MsgTemplating `json:"templating,omitempty"`
```

Comment:
Find the naming slightly awkward here but seems consistent with the action so I can convince myself it is ok.